collector/textfile.go

```diff
@@ -218,6 +219,27 @@ func (c *textFileCollector) Update(ch chan<- prometheus.Metric) error {
 			mtime, families, err := c.processFile(path, f.Name(), ch)
 
 			for _, mf := range families {
+				// Check for metrics with inconsistent help texts. In such a case:
+				// * update `node_textfile_scrape_error`,
+				// * update the last help text set with the differentiating help text (for each such instance), and,
+				// * log an error.
+				// Additionally, generate metric only for the first such instance (i.e., *mf.Help == helpTexts[0]).
```

Comment:
Overly verbose comment. We don't need to explain the code line by line.

```suggestion
				// Check for metrics with inconsistent help texts and take the first help text occurrence.
```

Reply:
ACK, changed. I'll chalk up the unnecessary verbosity to me being up late. 😄